doc/developer_guide/psyir_symbols.rst

```diff
@@ -220,6 +216,41 @@ the `__contains__` method has no mechanism to pass a `scope_limit`
 optional argument. This would probably require a separate `setter` and
 `getter` to specify whether to check ancestors or not.
 
+Copying Symbols and Symbol Tables
+=================================
+
+Since Symbols can contain PSyIR nodes and other Symbols (e.g. as part
+of the definition of their precision or initial value), creating copies
+is not entirely straightforward. Every `Symbol` has the `copy` method:
+
+.. automethod:: psyclone.psyir.symbols.Symbol.copy
+
+This ensures that the precision and initial-value properties are copied
+appropriately (e.g. new PSyIR nodes constructed) but any Symbols
+referred to by those properties remain unchanged.
+
+When performing a deep copy of a PSyIR tree, obviously all Symbols will
+need to be replaced with their equivalents in the new tree. The
+`SymbolTable.deep_copy()` method:
+
+.. automethod:: psyclone.psyir.symbols.SymbolTable.deep_copy
+
+handles this by first creating shallow copies of all Symbols in the
+table and then ensuring that each is updated to refer to Symbols in
+the new scope. This is achieved with the `replace_symbols_using`
+method:
+
+.. automethod:: psyclone.psyir.symbols.Symbol.replace_symbols_using
+
+All PSyIR `Node` classes also implement this method. In this case, it
```

Comment:
Remove "also"?

Reply:
I'd like to keep it to differentiate from the previous text which is about `Symbol` - it's slightly unusual for two, distinct classes to have the same method name.